lib/bulma_form_builder/inputs/select.rb

```diff
@@ -9,8 +9,10 @@ module Select
         def select_with_bulma(method, choices = nil, options = {}, html_options = {}, &block)
           html_options[:required] = options[:required]
 
+          div_class = ['select']
+          div_class.push('is-fullwidth') if options[:fullwidth]
```

Comment:
Is it possible to have `options[:fullwidth]` removed after this process?
Otherwise It will cause unnecessary options to be passed to `select_without_bulma`.

Reply:
yeah, sure.. good catch.